exceptions/src/commonMain/kotlin/Exceptions.kt

```diff
@@ -1,42 +1,42 @@
 package com.juul.kable
 
 /** Failure occurred with the underlying Bluetooth system. */
-public open class BluetoothException internal constructor(
```

Comment:
Is there currently a need to make the constructor public?

Reply:
Yes, since we also construct one as the superclass of `GattRequestRejectedException` which is still in `kable-core`